google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -346,6 +346,14 @@ public static BlobTargetOption metagenerationNotMatch() {
       return new BlobTargetOption(StorageRpc.Option.IF_METAGENERATION_NOT_MATCH);
     }
 
+    /**
+     *  Returns an option for blob's data disabledGzipContent. If this option is used,
+     *  the request will create a blob with disableGzipContent; at present, this is only for upload.
+     */
+    public static BlobTargetOption disableGzipContent() {
+      return new BlobTargetOption(StorageRpc.Option.IF_DISABLE_GZIP_CONTENT, true);
+    }
```

Comment:
Could you add a unit test for this option? 

Related unit tests: https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java#L406